docs/content/technical/build_process.md

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Build process information'
+weight: 35
+---
+
+## 1. Introduction
+
+NDS ROMs are complicated. They have an ARM9 binary, an ARM7 binary, an optional
```

Comment:
Writing "NDS ROMs are complicated." as the first sentence is going to be discouraging to people :-) Don't make them feel condescended to.

Reply:
FAir